pkg/model/record_test.go

```diff
@@ -1,107 +0,0 @@
-package model
```

Comment:
hey @msherif1234 , I'm removing this test because I think it is of quite low value: it only tests `ReadFrom`, which is really a direct call to `binary.Read`. Meaning all it does is testing that the `encoding/binary` standard lib is right. This, at a price of sometimes painful maintenance. Does it worth it?

Reply:
what is the reason u removing it because u removed packed so the structure no longer aligns ?

Reply:
the advantage of this make sure the auto generated structure are sane too

Reply:
I still don't get, what sort of issue could it catch, do you have an example?
Anyway, I'm fine reintroducing it. It was not only for the padding change, it's any change in the bpf types that make this test painful to maintain.

Reply:
Here we go: https://github.com/netobserv/netobserv-ebpf-agent/pull/469/commits/119061c05dbafe2dc07e45bd34419d3e58be23c2
But I'd still appreciate if you can give me a concrete example. It doesn't really test anything about the types.h / go struct mapping, it only test that a go struct is what it claims to be ... that doesn't make really sense to me.